### DISCUSSION TITLE
question about Colab ComfyUI Google Drive :  %CD inside mounted Drive like in the attached image? TY!

### DISCUSSION THREAD

**Discussion by Amit30swgoh** [Comfy-Org/ComfyUI-Manager#599] [General]
![image](https://github.com/ltdrdata/ComfyUI-Manager/assets/42430868/8633c96c-390d-495a-bab0-9a98ba2f91fd)

**Comment by ltdrdata** [comment]
I'm not sure exactly what you mean.
Once mounted, the drive will have a specific path, and it can be accessed using `%cd`.
That's all.

**Reply by Amit30swgoh** [reply]
ty, i meant to ask if %cd should not be inside comfyui btw friend is there a way to fix that both insightface and instantid will work without interferances of depandancies? ty so much!
